admin_guide/persistent_storage_nfs.adoc

```diff
@@ -84,3 +84,18 @@ reclamation of discarded resources can be configured for each persistent volume.
 Many ways to script these tasks exist.  We've created an link:https://github.com/openshift/openshift-ansible/tree/master/roles/kube_nfs_volumes[Ansible playbook] to help you get started.
 
 
+== SELinux and NFS export settings
+
+By default, SELinux will disallow a connection from a pod to a remote NFS server.  Enable this in SELinux on each node with the following command:
```

Comment:
It's not really true that the connection is disallowed. What I saw happen was that the pod wasn't allowed to write to an NFS mount. So without this, you would actually mount up the volume fine. The app would just fail when it tried to actually write some data.